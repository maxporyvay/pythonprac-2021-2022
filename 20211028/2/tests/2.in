print(*list(slide([[1],2,'',[2,3],(1,),None,'nlkjlj'], 3)))