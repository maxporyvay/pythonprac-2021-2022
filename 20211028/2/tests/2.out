[1] 2  2  [2, 3]  [2, 3] (1,) [2, 3] (1,) None (1,) None nlkjlj None nlkjlj nlkjlj
